system/authorized_key.py

```diff
@@ -460,6 +475,7 @@ def main():
            key_options = dict(required=False, type='str'),
            unique      = dict(default=False, type='bool'),
            exclusive   = dict(default=False, type='bool'),
+	   validate_certs = dict(default=True, type='bool'),
```

Comment:
you are right! do we have somehwere a compliance doc or something like that
do avoid this things in the future, is there some lint module to be ran
before commiting?

On Thu, Nov 26, 2015 at 1:28 PM, Will Thames <notifications@github.com>
wrote:

> In system/authorized_key.py
> <https://github.com/ansible/ansible-modules-core/pull/2519#discussion_r45973446>
> :
>
> > @@ -460,6 +475,7 @@ def main():
> >             key_options = dict(required=False, type='str'),
> >             unique      = dict(default=False, type='bool'),
> >             exclusive   = dict(default=False, type='bool'),
> > +	   validate_certs = dict(default=True, type='bool'),
>
> It's tabs. Line 126 and 478 have tabs rather than spaces. Please check
> your vim settings for softtabstop and expandtab.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ansible/ansible-modules-core/pull/2519/files#r45973446>
> .
>



-- 
Veaceslav Mindru
PNT Devops Sysops